universal_notifications/backends/emails.py

```diff
@@ -31,7 +32,7 @@ def send_email(template, to, subject, variables={}, fail_silently=False, cms=Fal
     replace_variables['protocol'] = protocol
     domain = variables['site'].domain
     replace_variables['domain'] = domain
-    for key, value in replace_variables.iteritems():
+    for key, value in (replace_variables.iteritems() if six.PY2 else replace_variables.items()):
```

Comment:
Just make it "items()" - it will work (less optimal, but will) on python 2.7. If possible it is best to avoid six, after all python 2.x should be dropped in two or three years